### ISSUE TITLE
[Bug][A11Y] Badge with icon or standard has no accessible name

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#385]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iOS / Android

### OS Version

.

### App version

0.6.0

### Describe the issue

Badge with icon or standard has no accessible name, 

### Expected Behavior

A badge should have a customisable accessible name has the objective is to give an information to the user ( notification, status...)

**Comment by Tayebsed93** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 441
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 441
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
